_data/companies.yml

```diff
@@ -936,13 +936,6 @@
   events: Restricted
   last_update: 2020-03-11
 
-- name: OneFit
-  wfh: Encouraged
-  travel: Restricted
-  visitors: Restricted
-  events: Restricted
-  last_update: 2020-03-12
-
```

Comment:
Looks like your git client tried to remove OneFit. Would you mind putting that back in?

Reply:
woah. no idea how this happened! i used the in line editor. will change

Reply:
Probably a race condition. I merged Tripadvisor in the last couple of minutes.